dev-debug/gdb/gdb-15.0.91.ebuild

```diff
@@ -93,6 +96,7 @@ RDEPEND="
 	python? ( ${PYTHON_DEPS} )
 	guile? ( >=dev-scheme/guile-2.0 )
 	xml? ( dev-libs/expat )
+	amd64? ( rocm? ( dev-libs/rocdbgapi ) )
```

Comment:
Why are we doing amd64? ( ...)?

Reply:
If I do without it, in `pkgcheck scan` I get:
```
  NonsolvableDepsInDev: version 14.2-r1: nonsolvable depset(depend) keyword(~arm) dev profile (default/linux/arm/23.0/armv4) (110 total): solutions: [ dev-libs/rocdbgapi ]
  NonsolvableDepsInDev: version 14.2-r1: nonsolvable depset(rdepend) keyword(~arm) dev profile (default/linux/arm/23.0/armv4) (110 total): solutions: [ dev-libs/rocdbgapi ]
  NonsolvableDepsInStable: version 14.2-r1: nonsolvable depset(depend) keyword(~arm) stable profile (default/linux/arm/23.0) (108 total): solutions: [ dev-libs/rocdbgapi ]
  NonsolvableDepsInStable: version 14.2-r1: nonsolvable depset(rdepend) keyword(~arm) stable profile (default/linux/arm/23.0) (108 total): solutions: [ dev-libs/rocdbgapi ]
```
as `dev-libs/rocdbgapi` is amd64-only right now.

Reply:
likely better to mask the USE then - in general, it is bad to have a USE that does nothing (especially if it does nothing only sometimes)

Reply:
@ArsenArsen , you mean like this? https://github.com/gentoo/gentoo/pull/37324/commits/11f4665e927b960efacd795e427e9f911466b4de

I added it to `package.use.stable.mask` too, given that `dev-libs/rocdbgapi` might be stable eventually.

Reply:
sigh.. gh did not associate my comment with the PR.  see https://github.com/gentoo/gentoo/commit/11f4665e927b960efacd795e427e9f911466b4de#r143890771

Reply:
Ah, thanks, I removed it from `package.use.stable.mask` then.